config/unicorn.rb

```diff
@@ -4,4 +4,4 @@ def load_file_if_exists(config, file)
 load_file_if_exists(self, "/etc/govuk/unicorn.rb")
 
 working_directory File.dirname(File.dirname(__FILE__))
-worker_processes 4
+worker_processes Integer(ENV.fetch("UNICORN_WORKER_PROCESSES", 3))
```

Comment:
@kevindew has been making some changes in this area recently, and for consistency, it might be good to switch to using the Unicorn configuration in newer versions of `govuk_app_config` [1]?

1: https://github.com/alphagov/govuk_app_config#unicorn

The effect would be different though, as the default is 2, rather than 3.

Reply:
Yeah, there's a bit more work involved than just bumping the gem.  The jasmine tests break when you bump to the new logstasher config, somewhere around the repointing of `stdout`.  I was hoping to avoid that here :-)

Reply:
ahh interesting! I was tinkering with frontend earlier didn't realise there was some pain with the bump of govuk_app_config and this. I'll be trying to sort it anyway for a docker change I'm trying to make.

Reply:
It's in https://github.com/alphagov/govuk-puppet/pull/7255 but I didn't want to merge it yet because I want to make sure frontend is OK first

Reply:
🎉 🎉 🌮